src/components/common/FormControl/FormControlTypes.tsx

```diff
@@ -0,0 +1,5 @@
+import { ReactElement } from 'react'
```

Comment:
For files where are stores types, we usually use file name `types.ts` or `types.tsx`. The name of the component is mentioned in parent folder name.

Reply:
Renamed.